lib/carrierwave-aws.rb

```diff
@@ -43,7 +44,33 @@ def self.normalized_acl(acl)
     normalized
   end
 
+  def self.aws_signer(value = nil)
+    self.aws_signer = value if value
```

Comment:
Yes, it's odd. But I tried to provide the same configuration interface as carrierwave does in its add_config magic [here](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/uploader/configuration.rb#L117).